src/backedges.jl

```diff
@@ -86,17 +86,17 @@ specTypes(mi::MethodInstance) = mi.specTypes
 instance(mi::MethodInstance) = mi
 nextnode(mi, edge) = edge
 
-instance(sfs::Vector{StackTraces.StackFrame}) = isempty(sfs) ? Core.Compiler.Timings.ROOTmi : sfs[end].linfo
+instance(sfs::Vector{StackTraces.StackFrame}) = isempty(sfs) ? Core.Compiler.Timings.ROOTmi : sfs[end].linfo::MethodInstance # we checked this type condition within `buildframes`
 method(sfs::Vector{StackTraces.StackFrame}) = method(instance(sfs))
 
 instance(ipframes::Vector{IPFrames}) = isempty(ipframes) ? Core.Compiler.Timings.ROOTmi : instance(ipframes[1].sfs)
 backedges(ipframes::Vector{IPFrames}) = (ret = ipframes[2:end]; isempty(ret) ? () : (ret,))
 
-function callstring(io, mi)
+function callstring(io::IO, mi::MethodInstance)
```

Comment:
Oops, this one was bad too (broke SnoopCompile). I will fix.

Reply:
Agh, sorry I just checked the result of the integration test.

Reply:
The mistake is mine. I just assumed we'd be fine, but I forgot that until #189 I didn't know how to test the interactive menus, and so SnoopCompile evidently doesn't have complete integration tests. I'll try to fix that, but meanwhile no hurry.